docs/data-modeling/command.mdx

```diff
@@ -79,17 +85,35 @@ ddn command add <connector_link_name> "*"
 
 This will add commands with their accompanying metadata definitions to your metadata.
 
-### Via native mutation {#via-native-mutation-how-to}
+### Via native operations {#via-native-operations-how-to}
+
+Some data connectors support the ability to add commands via native operations so that you can add any operation that is
+not supported by the automatic introspection process.
+
+For classic database connectors this will be native query code for that source. This can be, for example, a more complex
+read operation or to run some custom business logic and can be exposed as queries or mutations in the GraphQL API.
+
+For Lambda connectors, eg: (Typescript, Go, Python, etc) this will be a function (read-only) or procedure (mutation or
+other side-effects) that also be exposed as a query or mutation in the GraphQL API.
```

Comment:
```suggestion
For Lambda connectors, eg: (Typescript, Go, Python, etc) this will be a function (read-only) or procedure (mutation or
other side-effects) that can also be exposed as a query or mutation in the GraphQL API.
```